tinygrad/tensor.py

```diff
@@ -467,12 +467,13 @@ def conv2d(self, weight:Tensor, bias:Optional[Tensor]=None, groups=1, stride=1,
     ret = (x * weight.reshape(1, groups, rcout, *[1 for _ in range(len(oyx))], cin, *HW)).sum([-1-i for i in range(1+len(oyx))], keepdim=True).reshape(bs, cout, *oyx)
     return ret if bias is None else ret.add(bias.reshape(1, -1, *[1 for _ in range(len(HW))]))
 
-  def dot(self, w:Tensor) -> Tensor:
```

Comment:
Why did the letter change?

Reply:
Only because I need original shape later, just realized that it would be better to just move that condition into separate variable and compute at the beginning